app/views/judgment/judgmentBeforeYouUpload.scala.html

```diff
@@ -4,15 +4,20 @@
 
 @import views.html.helper.form
 
-@(consignmentId: UUID, warningMessage: String, name: String)(implicit request: RequestHeader, messages: Messages)
-@main("Transfer agreement", name = name) {
+@(consignmentId: UUID, name: String)(implicit request: RequestHeader, messages: Messages)
+@main("Before you upload", name = name) {
 @defining(play.core.PlayVersion.current) { version =>
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      @transferAgreementHeader()
-      <p class="govuk-body">Please confirm that the court judgment contains the following information.</p>
-      @transferAgreementWarning(warningMessage)
-      <p class="govuk-body">In case the court judgment you want to transfer has supplementary material: upload the court judgment using this service and email the supplementary material to
+      @progressIndicator(Messages("beforeYouUpload.progress"))
+      <h1 class="govuk-heading-l">Before you upload</h1>
+      <p class="govuk-body">Please check the court judgment contains the following information:
+        <strong>
+          Name(s) of judge(s), name of parties, court, judgment date, neutral citation.
+        </strong>
+        We cannot publish the court judgment without the information
```

Comment:
Should this not say: ... judgment without **this** information Might be worth double checking this?